<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="R01-CE62302732" failures="1" tests="3" name="Academy.ValidationNavigationHeader" time="0.645" errors="0" timestamp="2020-07-22T20:56:34 IST" skipped="0">
  <testcase classname="Academy.ValidationNavigationHeader" name="Header" time="0.473">
    <failure message="expected [The academy is l by Rahul Shetty, a Test evangelist and Test architect with over 10+ years in the IT industry.] but found [The academy is led by Rahul Shetty, a Test evangelist and Test architect with over 10+ years in the IT industry.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [The academy is l by Rahul Shetty, a Test evangelist and Test architect with over 10+ years in the IT industry.] but found [The academy is led by Rahul Shetty, a Test evangelist and Test architect with over 10+ years in the IT industry.]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at Academy.ValidationNavigationHeader.Header(ValidationNavigationHeader.java:77)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:41)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:443)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:67)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
    </failure>
  </testcase> <!-- Header -->
  <testcase classname="Academy.ValidationNavigationHeader" name="Nav" time="0.071"/>
  <testcase classname="Academy.ValidationNavigationHeader" name="stringMatch" time="0.101"/>
</testsuite> <!-- Academy.ValidationNavigationHeader -->
